docs/platforms/javascript/guides/aws-lambda/configuration/integrations/contextlines.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: ContextLines
+description: "Adds source code to your stack frames. (default)"
```

Comment:
@stephanie-anderson the reason I made these separate pages instead of solving this via `<PlatformSection>` in the common file is that the description has to be different - it is default for node, but not default for browser 😬 or is it possible to have a conditional description for certain guides? 🤔